src/declarativedbusinterface.cpp

```diff
@@ -426,6 +428,10 @@ void DeclarativeDBusInterface::pendingCallFinished(QDBusPendingCallWatcher *watc
 
     if (reply.isError()) {
         qmlInfo(this) << reply.error();
+        QJSValue errorCallback = callbacks.second;
+        if (errorCallback.isCallable()) {
+            errorCallback.call();
```

Comment:
Would be inconsistent usage of call(), no such checking with normal callback. Could have a separate commit changing both, but any idea how to test it?